src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,15 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String bladeSize;
```

Comment:
how can we access private field? 
add getter/setter
some for other model classes

Reply:
Fixed